UltimateTeam.Toolkit/Models/IListResponse.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace UltimateTeam.Toolkit.Models
+{
+    public interface IListResponse
```

Comment:
I see, thanks for explaining. I would prefer to keep the toolkit generic and not introduce too many specific interfaces etc. What i could do is create a base class for `AuctionResponse` and `WatchlistResponse` which has the same properties as the interface. Then you can use that in your method instead. Almost the same, but reduces some duplicated code and less confusing i think.

Would be great if you could push another commit to your branch that removes the interface.